dluhc-component-library/src/components/timetablePage/ProgressTile.tsx

```diff
@@ -0,0 +1,15 @@
+interface ProgressTileProps {
+  progress: "notStarted" | "delayed" | "inProgress" | "finished";
+}
+
+const ProgressTile = ({ progress }: ProgressTileProps) => {
+  const progressClass = `progress-${progress}`;
+
+  return (
+    <span className={`uppercase font-semibold py-1 px-2 ${progressClass}`}>
+      {progress}
```

Comment:
This value currently looks like "notStarted" so it needs to be converted to a human readable string